features/integration/applications.feature

```diff
@@ -84,3 +85,131 @@ Feature: Applications
          | program                     | state-location | global-bytes | local-bytes |
          | programs/loccheck.teal.tok  | local          | 0            | 1           |
          | programs/globcheck.teal.tok | global         | 1            | 0           |
+
+   @applications.boxes
+   Scenario: Exercise box features
+      # Make these tests 'standalone'.
+      # This should create a new, random account and save the public/private key for future steps to use.
+      Given I create a new transient account and fund it with 10000000000 microalgos.
+      # Create application
+      # depends on the transient account, and also the application id.
+      # Use suggested params
+      And I build an application transaction with the transient account, the current application, suggested params, operation "create", approval-program "programs/box_app.teal", clear-program "programs/box_app.teal", global-bytes 0, global-ints 0, local-bytes 0, local-ints 0, app-args "", foreign-apps "", foreign-assets "", app-accounts "", extra-pages 0, boxes ""
```

Comment:
nit: even though the proof is in the pudding that `programs/box_app.teal` works as the clear program is there a reason not to use the following more standard one instead?
```
#pragma version 8
int 1
```

Reply:
yea, I think it is time to make a `eight.teal` like https://github.com/algorand/algorand-sdk-testing/blob/feature/box-storage/features/resources/programs/six.teal

Reply:
resolved by #235